2021-03-13 08:44:11,579 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 1173)
2021-03-13 08:44:11,580 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 1173)
2021-03-13 08:44:21,622 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 1173)
2021-03-13 08:44:31,725 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 1173)
2021-03-13 08:44:32,136 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 1173)
2021-03-13 09:12:48,014 :: server.py:293 :: INFO :: Server exiting due to user input
2021-03-13 09:12:48,015 :: server.py:311 :: INFO :: User input thread joined
2021-03-13 09:12:48,962 :: server.py:313 :: INFO :: Server thread joined
2021-03-13 09:35:11,566 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2425)
2021-03-13 09:35:11,567 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2425)
2021-03-13 09:35:21,586 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2425)
2021-03-13 09:35:31,704 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2425)
2021-03-13 09:35:33,598 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2436)
2021-03-13 09:35:33,599 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2436)
2021-03-13 09:35:37,362 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2425)
2021-03-13 09:35:37,362 :: server.py:216 :: ERROR :: Malformed request received
2021-03-13 09:35:43,629 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2436)
2021-03-13 09:35:53,730 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2436)
2021-03-13 09:36:03,875 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2436)
2021-03-13 09:36:14,011 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2436)
2021-03-13 09:36:24,155 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2436)
2021-03-13 09:36:34,279 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2436)
2021-03-13 09:36:39,474 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2425)
2021-03-13 09:36:41,619 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2436)
2021-03-13 09:36:47,575 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2467)
2021-03-13 09:36:47,575 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2467)
2021-03-13 09:36:54,631 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2471)
2021-03-13 09:36:54,632 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:36:57,611 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2467)
2021-03-13 09:36:58,312 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2467)
2021-03-13 09:36:58,312 :: server.py:216 :: ERROR :: Malformed request received
2021-03-13 09:37:04,667 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:37:14,789 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:37:24,907 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:37:35,044 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:37:45,168 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:37:45,860 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2467)
2021-03-13 09:37:49,830 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2503)
2021-03-13 09:37:49,831 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2503)
2021-03-13 09:37:54,019 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2503)
2021-03-13 09:37:54,019 :: server.py:216 :: ERROR :: Malformed request received
2021-03-13 09:37:55,264 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:38:05,405 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:38:15,508 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:38:25,611 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:38:35,708 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:38:44,269 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2503)
2021-03-13 09:38:45,826 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:38:55,929 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:39:06,078 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:39:16,204 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:39:26,301 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:39:36,389 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:39:46,476 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:39:56,625 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:40:06,730 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:40:16,796 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:40:26,923 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:40:36,992 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:40:47,049 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:40:57,171 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:41:07,265 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:41:17,329 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:41:27,453 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:41:37,535 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:41:44,624 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2599)
2021-03-13 09:41:44,625 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2599)
2021-03-13 09:41:47,638 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:41:47,714 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2599)
2021-03-13 09:41:47,714 :: server.py:216 :: ERROR :: Malformed request received
2021-03-13 09:41:53,710 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2599)
2021-03-13 09:41:57,593 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2604)
2021-03-13 09:41:57,593 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2604)
2021-03-13 09:41:57,778 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:41:59,989 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2604)
2021-03-13 09:41:59,989 :: server.py:216 :: ERROR :: Malformed request received
2021-03-13 09:42:07,882 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:42:18,018 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:42:28,177 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:42:38,281 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:42:48,387 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:42:50,190 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2604)
2021-03-13 09:42:58,477 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:43:08,579 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:43:18,684 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:43:28,759 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:43:38,894 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:43:48,972 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:43:59,066 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:44:09,143 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:44:19,289 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:44:29,341 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:44:39,464 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:44:49,633 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:44:59,754 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:45:09,887 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:45:19,986 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:45:30,081 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:45:40,167 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:45:50,212 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:46:00,351 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:46:10,401 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:46:20,532 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:46:30,598 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:46:40,633 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:46:50,741 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:47:00,829 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:47:10,938 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:47:21,071 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:47:22,128 :: server.py:268 :: INFO :: Server exiting due to user input
2021-03-13 09:47:22,128 :: server.py:286 :: INFO :: User input thread joined
2021-03-13 09:47:31,154 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2471)
2021-03-13 09:47:31,562 :: server.py:288 :: INFO :: Server thread joined
2021-03-13 09:47:41,044 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2746)
2021-03-13 09:47:41,045 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2746)
2021-03-13 09:47:51,094 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2746)
2021-03-13 09:48:01,207 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2746)
2021-03-13 09:48:02,039 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2755)
2021-03-13 09:48:02,039 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2755)
2021-03-13 09:48:05,280 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2746)
2021-03-13 09:48:05,280 :: server.py:216 :: ERROR :: Malformed request received
2021-03-13 09:48:12,084 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2755)
2021-03-13 09:48:17,028 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2746)
2021-03-13 09:48:22,217 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2755)
2021-03-13 09:48:23,732 :: server.py:268 :: INFO :: Server exiting due to user input
2021-03-13 09:48:23,732 :: server.py:286 :: INFO :: User input thread joined
2021-03-13 09:48:25,855 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2755)
2021-03-13 09:48:26,117 :: server.py:288 :: INFO :: Server thread joined
2021-03-13 09:48:37,179 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2771)
2021-03-13 09:48:37,180 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:48:37,715 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2768)
2021-03-13 09:48:37,716 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:48:42,709 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:48:47,750 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:48:52,883 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:48:57,867 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:49:03,024 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:49:07,977 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:49:13,109 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:49:18,131 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:49:23,246 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:49:28,245 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:49:33,382 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:49:38,326 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:49:43,498 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:49:48,382 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:49:53,548 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:49:58,529 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:50:03,687 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:50:08,657 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:50:13,839 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:50:18,724 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:50:24,008 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:50:28,824 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:50:34,059 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:50:35,094 :: server.py:268 :: INFO :: Server exiting due to user input
2021-03-13 09:50:35,095 :: server.py:286 :: INFO :: User input thread joined
2021-03-13 09:50:38,944 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2768)
2021-03-13 09:50:44,155 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2771)
2021-03-13 09:50:45,051 :: server.py:288 :: INFO :: Server thread joined
2021-03-13 09:51:45,727 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2848)
2021-03-13 09:51:45,728 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2848)
2021-03-13 09:51:49,259 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2850)
2021-03-13 09:51:49,259 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:51:54,870 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2848)
2021-03-13 09:51:59,308 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:52:05,012 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2848)
2021-03-13 09:52:09,501 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:52:15,145 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2848)
2021-03-13 09:52:19,650 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:52:22,914 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2848)
2021-03-13 09:52:29,731 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:52:39,855 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:52:49,974 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:53:00,090 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:53:10,213 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:53:20,329 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:53:30,418 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:53:40,528 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:53:50,657 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:53:56,581 :: server.py:268 :: INFO :: Server exiting due to user input
2021-03-13 09:53:56,581 :: server.py:286 :: INFO :: User input thread joined
2021-03-13 09:54:00,733 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2850)
2021-03-13 09:54:53,653 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2925)
2021-03-13 09:54:53,654 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:54:57,216 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 2927)
2021-03-13 09:54:57,217 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:55:00,785 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:55:07,235 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:55:11,004 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:55:17,357 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:55:21,109 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:55:27,468 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:55:31,232 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:55:37,594 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:55:41,314 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:55:47,726 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:55:51,454 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:55:57,878 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:56:01,609 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:56:08,001 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:56:11,794 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:56:18,098 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:56:21,939 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:56:28,169 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:56:32,089 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:56:38,270 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:56:42,212 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:56:48,387 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:56:52,334 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:56:58,529 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:57:02,474 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:57:08,624 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:57:12,587 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:57:18,772 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:57:22,724 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:57:28,862 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:57:32,890 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:57:38,967 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:57:43,011 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:57:49,100 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:57:53,125 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:57:59,217 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:58:02,624 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:58:09,297 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:58:12,762 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:58:19,457 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:58:22,882 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:58:29,580 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:58:32,999 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:58:39,671 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:58:43,116 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:58:49,784 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:58:53,196 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:58:59,897 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:59:03,298 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:59:09,965 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:59:13,362 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:59:20,038 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:59:23,489 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:59:30,187 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:59:33,589 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:59:40,345 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:59:43,686 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 09:59:50,449 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 09:59:50,947 :: server.py:271 :: INFO :: Server exiting due to user input
2021-03-13 09:59:50,947 :: server.py:289 :: INFO :: User input thread joined
2021-03-13 09:59:53,797 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2925)
2021-03-13 10:00:00,587 :: server.py:164 :: INFO :: Received message from ('127.0.0.1', 2927)
2021-03-13 10:00:01,175 :: server.py:291 :: INFO :: Server thread joined
2021-03-13 10:01:12,113 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3095)
2021-03-13 10:01:12,114 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3095)
2021-03-13 10:01:13,548 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3096)
2021-03-13 10:01:13,549 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3096)
2021-03-13 10:01:22,154 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3095)
2021-03-13 10:01:23,586 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3096)
2021-03-13 10:01:32,279 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3095)
2021-03-13 10:01:33,691 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3096)
2021-03-13 10:01:42,429 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3095)
2021-03-13 10:01:43,773 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3096)
2021-03-13 10:01:52,571 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3095)
2021-03-13 10:01:53,903 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3096)
2021-03-13 10:02:02,679 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3095)
2021-03-13 10:02:03,987 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3096)
2021-03-13 10:02:08,376 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3096)
2021-03-13 10:02:10,292 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3095)
2021-03-13 10:02:15,092 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3119)
2021-03-13 10:02:15,094 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3119)
2021-03-13 10:02:20,130 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3122)
2021-03-13 10:02:20,130 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3122)
2021-03-13 10:02:24,824 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3119)
2021-03-13 10:02:30,161 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3122)
2021-03-13 10:02:35,010 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3119)
2021-03-13 10:02:37,498 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3119)
2021-03-13 10:02:40,274 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3122)
2021-03-13 10:02:47,565 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3119)
2021-03-13 10:02:50,391 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3122)
2021-03-13 10:02:57,638 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3119)
2021-03-13 10:03:00,470 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3122)
2021-03-13 10:03:07,736 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3119)
2021-03-13 10:03:10,584 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3122)
2021-03-13 10:03:14,038 :: server.py:288 :: INFO :: Server exiting due to user input
2021-03-13 10:03:14,039 :: server.py:306 :: INFO :: User input thread joined
2021-03-13 10:03:17,885 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3119)
2021-03-13 10:03:20,657 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3122)
2021-03-13 10:03:21,309 :: server.py:308 :: INFO :: Server thread joined
2021-03-13 10:04:14,543 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3164)
2021-03-13 10:04:14,544 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3164)
2021-03-13 10:04:14,675 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3166)
2021-03-13 10:04:14,675 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:04:24,587 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3164)
2021-03-13 10:04:24,723 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:04:25,160 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:04:34,709 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3164)
2021-03-13 10:04:35,313 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:04:44,802 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3164)
2021-03-13 10:04:45,366 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:04:46,839 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:04:54,879 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3164)
2021-03-13 10:04:56,963 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:05:02,969 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:05:04,989 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3164)
2021-03-13 10:05:13,062 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:05:15,086 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3164)
2021-03-13 10:05:23,207 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:05:25,192 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3164)
2021-03-13 10:05:28,752 :: server.py:288 :: INFO :: Server exiting due to user input
2021-03-13 10:05:28,752 :: server.py:306 :: INFO :: User input thread joined
2021-03-13 10:05:33,339 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3166)
2021-03-13 10:05:35,283 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3164)
2021-03-13 10:05:35,903 :: server.py:308 :: INFO :: Server thread joined
2021-03-13 10:06:11,207 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3209)
2021-03-13 10:06:11,208 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3209)
2021-03-13 10:06:11,563 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3207)
2021-03-13 10:06:11,564 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3207)
2021-03-13 10:06:14,573 :: server.py:288 :: INFO :: Server exiting due to user input
2021-03-13 10:06:14,574 :: server.py:306 :: INFO :: User input thread joined
2021-03-13 10:06:21,242 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3209)
2021-03-13 10:06:21,598 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3207)
2021-03-13 10:06:22,594 :: server.py:308 :: INFO :: Server thread joined
2021-03-13 10:06:45,790 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3223)
2021-03-13 10:06:45,791 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3223)
2021-03-13 10:06:54,816 :: server.py:65 :: INFO :: Connected to ('127.0.0.1', 3227)
2021-03-13 10:06:54,817 :: server.py:116 :: INFO :: Received message from ('127.0.0.1', 3227)
2021-03-13 10:06:55,811 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3223)
2021-03-13 10:07:04,860 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3227)
2021-03-13 10:07:05,936 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3223)
2021-03-13 10:07:11,140 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3227)
2021-03-13 10:07:14,556 :: server.py:165 :: INFO :: Received message from ('127.0.0.1', 3223)
2021-03-13 10:07:25,427 :: server.py:288 :: INFO :: Server exiting due to user input
2021-03-13 10:07:25,427 :: server.py:306 :: INFO :: User input thread joined
2021-03-13 10:07:25,912 :: server.py:308 :: INFO :: Server thread joined
